### ISSUE TITLE
After login expires, login button does not reappear

### ISSUE THREAD

**Issue by sisch** [ludumdare/ludumdare#425]
I left the theme voting page open in my browser and the login expired. Upper right corner was still showing user Icon. (1)

What I expected to happen:
Clicking on the user icon should have brought me to login again. Or clicking on any sub page link should have refreshed the navbar.

What happened(2):
It did show my user page as seen from someone else. Navbar did not refresh.

What resolved this problem for me:
Reloading the whole page obviously brought back the login button.

(1)<img width="1484" alt="theme voting page with expired login" src="https://cloud.githubusercontent.com/assets/5693975/20961193/4c0a63c4-bc64-11e6-82ea-88adb79fa970.png">
(2)<img width="1409" alt="user page with expired login" src="https://cloud.githubusercontent.com/assets/5693975/20961194/4c0a844e-bc64-11e6-8a94-ec8e3f3fd3ac.png">

**Comment by povrazor** [comment]
Yep, confirmed. I'm only partially done fixing this (all requests now tell you if you're logged in or not, I just don't do anything about it).

**Comment by povrazor** [comment]
Partially mitigated, thanks to sessions being fixed.
